core/src/main/java/com/backblaze/b2/json/B2Json.java

```diff
@@ -462,6 +462,17 @@ public String listToJson(List<?> list, Class<?> valueClass, B2JsonOptions option
         String typeField();
     }
 
+    /**
+     * <p>Class annotation that applies to a class that is a @union.</p>
+     *
+     * <p>The value provided when de-serializing and the type is unknown.</p>
+     */
+    @Retention(RetentionPolicy.RUNTIME)
+    @Target(ElementType.TYPE)
+    public @interface defaultForUnknownType {
```

Comment:
it's less a "default" and more a "replacementValue"....but this is completely consistent with B2Json.defaultForInvalidEnumValue, so....i guess it's fine.